packages/components/src/slider/slider/hook.ts

```diff
@@ -0,0 +1,131 @@
+/**
+ * WordPress dependencies
+ */
+import { useCallback, useMemo, useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import * as styles from '../styles';
+import { useContextSystem, WordPressComponentProps } from '../../ui/context';
+import { useFormGroupContextId } from '../../ui/form-group';
+import { useControlledValue, useCx } from '../../utils/hooks';
+import { interpolate } from '../../utils/interpolate';
```

Comment:
Same comment as the `flow` utility (once it's exported from `utils` folder):

```suggestion
import { interpolate } from '../../utils';
```

Reply:
I'd be happy to add this to the `utils` exports. Before I do, I was [wondering](https://github.com/WordPress/gutenberg/pull/42315#discussion_r926318120) if we should be exporting all the math util functions and if this should then be part of that?

Reply:
Good point. Marking this as resolved in favour of https://github.com/WordPress/gutenberg/pull/42315#discussion_r925670265